woocommerce/woocommerce
# Pass `WC_ADMIN_PHASE=core` to build commands & Remove "plugin" env

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32618.

- Pass `WC_ADMIN_PHASE=core` to build commands to build code in the core phrase.
 - Remove "plugin" env from feature plugin config


### How to test the changes in this Pull Request:

1. Run `pnpm nx clean woocommerce-admin`
2. Run `pnpm nx build woocommerce`
3. Confirm that `plugins/woocommerce/includes/react-admin/feature-config.php` is generated from `core.json` (`minified-js => false`).
4. Observe that there are no `*.min.js` files in `plugins/woocommerce/assets/client/admin/app/` and `plugins/woocommerce/assets/client/admin/app/index.js` is minified.


### Other information:




### FOR PR REVIEWER ONLY:
